Migrate to new conan center remote

```
-- Conan executing: /usr/local/bin/conan install /__w/vulkan-renderer/vulkan-renderer --remote conan-center --build outdated --options build_benchmarks=False --options build_tests=False --profile default --settings build_type=Release --settings compiler=clang --settings compiler.version=12 --settings compiler.libcxx=libstdc++11 --settings compiler.cppstd=17 --settings compiler.libcxx=libstdc++11
WARN: Remotes registry file missing, creating default one in /github/home/.conan/remotes.json
ERROR: No remote 'conan-center' defined in remotes
CMake Error at build/conan.cmake:631 (message):
  Conan install failed='1'
Call Stack (most recent call first):
  cmake/conan_setup.cmake:52 (conan_cmake_install)
  CMakeLists.txt:38 (include)
```

--- a/cmake/conan_setup.cmake
+++ b/cmake/conan_setup.cmake
@@ -36,7 +36,7 @@ if(${is_multi_config} AND NOT DEFINED CMAKE_BUILD_TYPE)
         conan_cmake_install(
             PATH_OR_REFERENCE ${PROJECT_SOURCE_DIR}
             BUILD outdated
-            REMOTE conan-center
+            REMOTE conancenter
             PROFILE ${INEXOR_CONAN_PROFILE}
             GENERATOR cmake_multi
             OPTIONS build_benchmarks=${benchmark_option}
@@ -52,7 +52,7 @@ else()
     conan_cmake_install(
         PATH_OR_REFERENCE ${PROJECT_SOURCE_DIR}
         BUILD outdated
-        REMOTE conan-center
+        REMOTE conancenter
         PROFILE ${INEXOR_CONAN_PROFILE}
         OPTIONS build_benchmarks=${benchmark_option}
         OPTIONS build_tests=${tests_option}

